recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
```

Comment:
Please use jinja templating variables as shown in the example:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L6

Reply:
done